lib/blocks/latest-posts.php

```diff
@@ -44,19 +44,25 @@ function gutenberg_render_block_core_latest_posts( $attributes ) {
 		$post_id = $post['ID'];
 		$post_permalink = get_permalink( $post_id );
 		$post_title = get_the_title( $post_id );
+		$post_the_date = get_the_date( false, $post_id );
+		$post_date = '';
 
-		$posts_content .= "<li><a href='{$post_permalink}'>{$post_title}</a></li>\n";
+		if ( $attributes['displayPostDate'] ) {
+			$post_date = "<span class='wp-block-latest-posts__post-date'>{$post_the_date}</span>";
+		}
+
+		$posts_content .= "<li><a href='{$post_permalink}'>{$post_title}</a>{$post_date}</li>\n";
```

Comment:
The return value of `get_permalink()` also needs to go through `esc_url()`, which was missed previously.

The return values of `get_the_date()` and `get_the_title()` should probably also go through `esc_html()`.